packages/block-library/src/table-of-contents/save.js

```diff
@@ -13,8 +14,13 @@ export default function save( { attributes: { headings = [] } } ) {
 	if ( headings.length === 0 ) {
 		return null;
 	}
+
+	const blockProps = useBlockProps.save( {
+		'aria-label': __( 'Table of Contents' ),
```

Comment:
This allows us to pass an accessible name to the front-end.

Reply:
We can't use translatable strings in the `save` method. It breaks the "purity" of a function, as it can return different values when the same arguments are passed based on user locale. This breaks block validation.

P.S. I'm in the process of significant refactoring of a ToC block, which will unblock this issue. Ref: #54224.

Reply:
@Mamaduka Had no idea. Why don't we have a static check for that? I can remove the translation, then it should be okay?

Reply:
Not sure, but that’s a good idea.

You can ignore saved content for now. The block will be dynamic after I land my refactoring and I’ll include the label.

Reply:
@Mamaduka Oh, that's cool. You are building a PHP renderer for the block. Sounds good. I will get rid of this change. 👍

Reply:
Thank you, @alexstine!

Reply:
Done. 👍